package.json

```diff
@@ -18,50 +18,49 @@
   ],
   "scripts": {
     "bounce": "rm -rf /node_modules && yarn",
```

Comment:
What does this do? Does it like clear cache or delete something?

Reply:
in a sense, yea.  I probably should just remove this cuz it's not a frequent thing I use.  It removes the installed node modules and redownloads them based on the yarn.lock file.  It's useful when you could have mismatched installed packages or tampered packages/files.